libraries/AP_Motors/AP_MotorsHeli_Single.h

```diff
@@ -140,9 +140,10 @@ class AP_MotorsHeli_Single : public AP_MotorsHeli {
     AP_Int16        _tail_type;                 // Tail type used: Servo, Servo with external gyro, direct drive variable pitch or direct drive fixed pitch
     AP_Int16        _ext_gyro_gain_std;         // PWM sent to external gyro on ch7 when tail type is Servo w/ ExtGyro
     AP_Int16        _ext_gyro_gain_acro;        // PWM sent to external gyro on ch7 when tail type is Servo w/ ExtGyro in ACRO
-    AP_Float        _collective_yaw_effect;     // Feed-forward compensation to automatically add rudder input when collective pitch is increased. Can be positive or negative depending on mechanics.
+//    AP_Float        _collective_yaw_effect;     // Feed-forward compensation to automatically add rudder input when collective pitch is increased. Can be positive or negative depending on mechanics.
```

Comment:
we can just delete this commented out line I think.

Reply:
@rmackay9 Thanks for catching those.  I had forgot that I just commented this out.  I was trying to quickly get something out for a customer.  I will take care of it.

Reply:
fixed. thanks!